sql/project_record_ids.sql

```diff
@@ -25,8 +25,32 @@ DROP TABLE IF EXISTS project_record_ids;
 SELECT * INTO project_record_ids 
 FROM _project_record_ids;
 
+
+
+/* Add stand-alone projects. This includes unmatched residential DOB projects, 
+as well as projects from sources that were excluded 
+from the non-DOB match process. */
+INSERT INTO project_record_ids
+SELECT array[]::text[]||record_id as project_record_ids
+FROM (
+	SELECT record_id::text from combined
+	WHERE (no_classa = '0' OR no_classa IS NULL) 
+	AND source <> 'DOB'
```

Comment:
Hey @mbh329  see the adjustments here I made for addressing your comments about that drop duplicates question you had. It turns out I added bunch of irrelevant dob records here earlier by not excluding the non-cluster DOB records. That's why I had to do the drop duplicates the end but that is not a real solution for the problem. With this new logic it should be good to go. Let me know if you still have any question about this